docroot/modules/custom/foia_webform/src/Plugin/WebformHandler/FoiaEmailWebformHandler.php

```diff
@@ -101,7 +101,7 @@ public function sendMessage(WebformSubmissionInterface $webform_submission, arra
    *   Returns the array as a comma separated string.
    */
   public function arrayToString(array $data) {
-    $handle = fopen('php://temp', 'c');
+    $handle = fopen('php://temp', 'w');
```

Comment:
Wondering why we changed this back to `w`. I thought you had looked at the documentation for `fopen` and found that `c` would work better for us?

Reply:
In theory <c> would be better, but the test didn't pass, so I changed it back to <w>.